### ISSUE TITLE
[Suggestion] One courier hut per warehouse

### ISSUE THREAD

**Issue by lebanni** [ldtteam/minecolonies-features#538]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [x] I made sure that **this change request is not a duplicate of any existing change request.**

### Context 

- Minecraft Version: 1.16.5
- MineColonies Version: 1.0.8
- Related Mods and their Versions:

## Is your feature request related to a problem?

Currently, we need 10 courier hut for a level 5 warehouse, requiring a lot a space.

## Enhancement or Change description

What about the possibility to recruit 2 workers for each level of a courier hut to save space ?

### Notes or related things

---
## Viewers

* Add a thumbs-up to the change request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by warning-djkajslh** [comment]
Approve, I've never like that you can't directly hire Couriers from the warehouse but you need to have tons of spaces for the courier huts. The couriers never even tend to the huts either, therefore the huts are a waste of space and resources.
This would mean all courier huts schematic becomes useless and obsolete though, but we can still get behind that dumb restriction if we want to, can't we.

**Comment by SivartMcDorf** [comment]
agree, i hate having to build huts everywhere taking up massive amounts of space.  I have heard of some people putting them underground just to avoid having to see them since the couriers don't actually need to use them.

**Comment by wklarper** [comment]
Agreed, although I don't think it is as bad as some believe. What I would like is that the courier hut provided bed space.  However, in lieu of that, I just create a schematic for a very small "kiosk".  I like to think of it as a valet stand. I build 1 courier hut as normal, then several of my kiosks as needed. The kiosk is  a courier hut with a very small footprint. I make levels that blend in with the style of the colony and place where needed. It gives the feel of a "main office" and several worker stands, and uses the mod as intended.

**Comment by MotionlessTrain** [comment]
Several styles have small courier's huts without beds, often built into the warehouse.
That is one of the reasons why the courier's hut can't become housing

**Comment by wklarper** [comment]
Guard towers provide housing without a bed requirement.  No reason the courier hut couldn't do the same. However, the realestate problem of the courier hut taking up too much space can be mitigated within the current mod in several ways.

**Comment by MotionlessTrain** [comment]
There is one big difference: guards don't sleep, they just nap every so often (they need to guard during the night too).
Couriers do. (Or otherwise they will make mistakes because of sleep deprivation)

Note that the combat academy and archery count as housing too, and they actually have beds as part of the requirements. Courier's huts didn't, and it would be too hard to change all courier huts now to include a bed

**Comment by wklarper** [comment]
Agreed. All I am saying is that the courier hut provides nothing except the job slot, kind of like a guard tower.  Other huts provide resources as well as the job slot. So, a house is always needed for both a courier job and any other non-military job, but the courier hut seems a bit superfluous with the advent of the warehouse, which is what the OP was addressing.
